en/frames_multicopter/dji_f450_cuav_5plus.md

```diff
@@ -0,0 +1,212 @@
+12C Expansion Board      | 1 | 1
+TTL Plate                | 1 | 1
+NEO GPS                  | - | 1
+GPS Bracket              | - | 1
+
+### Electronics
+
+Description | Quantity
+--- | --- 
+Motors - DR2205 KV2300             | 4
+Fully assembled Power Management Board with ESCs | 4
+Holybro power module               | 1
+Fr-sky D4R-II receiver             | 1
+CUAV V5+                           | 1
+Holybro GPS Neo-M8N                | 1
+Battery lumenier 1300 mAh 4S 14.8V | 1
+Vtx 5.8gHz                         | 1
+FPV camera                         | 1
+
+### Tools needed
+
+The following tools are used in this assembly:
+
+- 2.0mm Hex screwdriver
+- 3mm Phillips screwdriver
+- Wire cutters
+- Precision tweezers
+- Soldering iron
+
+
+![Required tools](../../assets/airframes/multicopter/dji_f450_cuav_5plus/required_tools.jpg)
+
+## Assembly
+
+Estimated time to assemble is approximately 90 minutes (about 45 minutes for the frame and 45 minutes installing the autopilot and configuring the airframe.
+
+1. Attach the 4 arms to the bottom plate using the provided screws.
+
+   ![Arms to bottom plate](../../assets/airframes/multicopter/dji_f450_cuav_5plus/1_attach_arms_bottom_plate.jpg)
+
+1. Solder ESC (Electronic Speed Controller) to the board, positive (red) and negative (black).
+
+   ![Solder ESCs](../../assets/airframes/multicopter/dji_f450_cuav_5plus/2_solder_esc.jpg)
+
+1. Solder the Power Module, positive (red) and negative (black).
+
+   ![Solder power module](../../assets/airframes/multicopter/dji_f450_cuav_5plus/3_solder_power_module.jpg)
+
+1. Plug in the motors to the ESCs according to their positions.
+
+   ![Plug in motors](../../assets/airframes/multicopter/dji_f450_cuav_5plus/4_plug_in_motors.jpg)
+
+1. Attach the motors to the corresponding arms.
+
+   ![Attach motors to arms (white)](../../assets/airframes/multicopter/dji_f450_cuav_5plus/5a_attach_motors_to_arms.jpg)
+   ![Attach motors to arms (red)](../../assets/airframes/multicopter/dji_f450_cuav_5plus/5b_attach_motors_to_arms.jpg)
+
+1. Add the top board (screw into the top of the legs).
+
+   ![Add top board](../../assets/airframes/multicopter/dji_f450_cuav_5plus/6_add_top_board.jpg)
+
+1. Add double-sided tape to the CUAV V5+ flight controller (it has internal vibration damping, so no need to use foam).
+
+   ![Damping foam](../../assets/airframes/multicopter/dji_f450_cuav_5plus/7_attach_cuav5plus.jpg)
+
+1. Attach the FrSky receiver to the bottom board with double-sided tape.
+
+   ![Attach FrSky receiver with double-sided tape](../../assets/airframes/multicopter/dji_f450_cuav_5plus/8_attach_frsky.jpg)
+
+1. Attach the telemetry module to the vehicle’s bottom board using double-sided tape.
+
+   ![Attach telemetry radio](../../assets/airframes/multicopter/dji_f450_cuav_5plus/9a_telemtry_radio.jpg)
+   ![Attach telemetry radio](../../assets/airframes/multicopter/dji_f450_cuav_5plus/9b_telemtry_radio.jpg)
+
+1. Put the aluminium standoffs on the button plate.
```

Comment:
@tony3dr
- Is this button plate, or bottom plate?
- Do we have any images, or need them?